forge/test/operators/utils/compat.py

```diff
@@ -22,6 +22,85 @@ class TestDevice:
     pass
 
 
+class TestTensorsUtils:
+
+    dev_data_format_to_dtype = {
+        # forge.DataFormat.Bfp2: torch.float16,
+        forge.DataFormat.Bfp2: torch.bool,  # mapped
+        forge.DataFormat.Bfp2_b: torch.bfloat16,
```

Comment:
Isn't this bool as well?

Reply:
I would say contrary... As stated above, it seems illogical to cast any float to boolean...

Reply:
Here was by mistake selected Bfp2 instead of Bfp2_b.

In general mapping of datatypes here is done according to forge.tensor.pytorch_dtype_to_forge_dataformat. E.x. https://github.com/tenstorrent/tt-forge-fe/blob/589e2ed6251228016ab3fa6fcd093f9a7b5f3687/forge/forge/tensor.py#L604

Reply:
No booleans for now